packages/patternfly-3/patternfly-react-extensions/less/vertical-tabs.less

```diff
@@ -41,4 +37,40 @@
   &:first-of-type {
     margin-top: 0;
   }
+
+  > .vertical-tabs-pf {
+    > .vertical-tabs-pf-tab {
+      position: initial;
+      padding-left: 15px;
+    }
+  }
+}
+
+.vertical-tabs-pf.restrict-tabs {
+  .vertical-tabs-pf-tab {
+    display: none;
+
+    /* Show any active tab, tab that has an active descendant, or is force shown */
+    &.active,
+    &.active-descendant,
+    &.shown {
+      display: block;
+    }
+  }
+
+  /* Show siblings of the active tab */
```

Comment:
should this comment be something like "Show tabs that are a direct descendent of the active tab"?

Reply:
This is trying to show tabs at the same level as the active tab not the active tab's descendants. Maybe `active-tab` is not clearly named. Maybe `has-active-tab` would be better. This selector implies that the grouping contains the active-tab as a direct child.

Reply:
oh ok, gotcha.